### PR TITLE
CIV-5023 Hearing Notice & Evidence Upload release

### PR DESCRIPTION
**Pull request by m-meulendijks-v1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5023

### Change description ###
Hearing Notice & evidence upload release ccd config

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-EvidenceUploadJudge-nonprod.json
Comment (sankhajuria) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-EvidenceUploadJudge-nonprod.json:1]: Its should be candidate for release, why its in nonprod ?

File: ccd-definition/AuthorisationCaseField/HearingNotice.json
Comment (sankhajuria) [inline ccd-definition/AuthorisationCaseField/HearingNotice.json:1]: Evidence upload and hearing notice can be move to the base authorisation, why we created a new JSON configuration?

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HearingNotice.json
Comment (sankhajuria) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HearingNotice.json:1]: Both Hearing notice and evidence upload can be merged to one file then we can merge it to the base file

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-EvidenceUploadJudge-nonprod.json
Comment (m-meulendijks-v1) [reply ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-EvidenceUploadJudge-nonprod.json:1]: My understanding is that it is not a candidate for release.
My understanding is that EvidenceUploadJudge is the 'Add Case Note' feature which was removed from the release as Doug confirmed this morning.

**Pull request closed without merging**